src/backend/marsha/core/utils/medialive_utils.py

```diff
@@ -284,6 +284,14 @@ def create_live_stream(key):
     }
 
 
+def wait_medialive_channel_is_created(channel_id):
+    """Wait until medialive channel is created."""
+    input_waiter = medialive_client.get_waiter("channel_created")
+    input_waiter.wait(
```

Comment:
so this is  block a gunicorn worker?

Reply:
Yes

Reply:
Then maybe we shouldn't wait and either poll from the client or push a notification via xmpp?

Reply:
Then we have to split (again) the create and start in two separated endpoints. One for creating and an other one to start the live. And introduce new state I think.

Reply:
I'm looking to split what I made in multiple endpoints

Reply:
As discussed let's postpone this refactoring to a future PR.